app/controllers/tasks_controller.rb

```diff
@@ -1,14 +1,24 @@
-# frozen_string_literal: true
 
 class TasksController < ApplicationController
   before_action :set_task, only: %i[show edit update destroy]
 
   # GET /tasks
   # GET /tasks.json
   def index
-    @tasks = Task.all
+    puts params['complete']
+    if params['completed']=='true'
+      @tasks=Task.where(completed: true)
+      @type='Completed Tasks'
```

Comment:
How were you planning to use this variable? I noticed it's not referenced in the code elsewhere.

Reply:
Good catch - I think I was originally going to use that variable. But, decided to not use it anymore. I can remove it.